EIPS/eip-6774.md

```diff
@@ -0,0 +1,262 @@
+---
+eip: 6774
+title: Non-Sellable Tokens
+description: Prevent speculation on NFTs by replacing transfers by a barter mechanism
+author: Cédric Nicolas (@cedric-n-icolas), Casey <casey@logion.community>, Kerel Verwaerde (@NSTKerel), Matthieu Chassagne <Matthieu@amcconsults.com>, Perrin (@pgrandne), Rafael Fitoussi (@fi2c), Raphael (@RaphaelHardFork), Virgil (@virgilea2410)
+discussions-to: https://ethereum-magicians.org/t/eip-6774-non-sellable-tokens/13528
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-03-15
+requires: 712, 721
+---
+
+## Abstract
+
+This EIP is an extension of [ERC-721](./eip-721.md) to create non-sellable tokens (NST). It proposes a strong restriction on transfers in order to prevent speculation on token price while maintaining the possibility of transfers, transfers are performed in a barter way (send to receive or receive to send) in an ecosystem controlled by the creator.
+
+## Motivation
+
+Interest for soulbound tokens (SBT) in the Ethereum community still growing since the V. Buterin idea publication (Soulbound, 2022-01-26), highlighting the non-transferability (non-)features to prevent, _in fine_, speculation of the token. While a lot of propositions emerged for implementation of SBT, the non-transferability is often too restrictive and requires a certain level of centralization, especially when users want to transfer a SBT between two owned accounts. Thus SBTs are way more fitted for account-bounded properties such as reputation, voting rights, privileges, ...
+
+In case of transferable items which cannot be sellable, SBTs can fit, but they require a strong level of centralization as transfer are restricted by the token creator (or the community in case of DAO managed SBT). Here some examples of possibles NSTs:
```

Comment:
```suggestion
In case of transferable items which cannot be sold, SBTs can fit, but they require a strong level of centralization as transfer are restricted by the token creator (or the community in case of DAO managed SBT). Here some examples of possible NSTs:
```